config/private_key.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	vault "github.com/hashicorp/vault/api"
+	"github.com/iotexproject/iotex-core/blockchain"
+	"github.com/pkg/errors"
+	uconfig "go.uber.org/config"
+)
+
+const defaultHTTPTimeout = 10 * time.Second
+
+type (
+	privKeyConfig struct {
+		ProducerPrivKey string `yaml:"producerPrivKey"`
```

Comment:
no need for this one, since `Config` already has it?

Reply:
https://github.com/iotexproject/iotex-core/issues/3413 
fix this issues. keep `Config . ProducerPrivKey ` for compatible